scripts/docker-entrypoint.sh

```diff
@@ -107,7 +107,7 @@ if [[ "$custom_artifactory_auth" == "" && ( "$SECRET_CUSTOM_ARTIFACTORY_APIKEY_H
         exit 1
     else
         echo "Using API key for authentication of custom artifactory to download JDBC driver."
-        custom_artifactory_auth="-H \"${SECRET_CUSTOM_ARTIFACTORY_APIKEY_HEADER}:${SECRET_CUSTOM_ARTIFACTORY_APIKEY}\""
+        custom_artifactory_auth="-H "$SECRET_CUSTOM_ARTIFACTORY_APIKEY_HEADER":"$SECRET_CUSTOM_ARTIFACTORY_APIKEY
```

Comment:
maybe I'm missing something, but is it intended that second string isn't closed?